src/connected/autobinding.ts

```diff
@@ -118,7 +118,10 @@ export class AutoBindingService {
       sqConnections
     );
     const connectionToBestHits = getBestHitsForConnections(connectionToServerProjects, unboundFolder);
-    this.promptToAutoBind(connectionToBestHits, unboundFolder);
+    if (connectionToBestHits.size > 0) {
+      // We don't want to show notification if it is not actionable
```

Comment:
There will be no matches if there is no successful connection + if there are really no matches